app/definition/ITeam.js

```diff
@@ -1,5 +1,5 @@
 // https://github.com/RocketChat/Rocket.Chat/blob/develop/definition/ITeam.ts
-export const TEAM_TYPE = {
+exports.TEAM_TYPE = {
```

Comment:
Why?

Reply:
Caused an error on local node version while running the tests Could also define minimum Node version instead.